dependencies/config/config_test.go

```diff
@@ -42,7 +42,7 @@ func init() {
 	hostname, err = os.Hostname()
 
 	if err != nil {
-		fmt.Errorf("Could not get hostname", err)
+		_ = fmt.Errorf("Could not get hostname %v", err)
```

Comment:
Maybe better to remove the whole if statement and change it to:
```
Expect(err).NotTo(HaveOccurred())
```
?

Reply:
I agree. Fixed